modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_logical-volume-manager-storage"]
+= Integrating software RAID arrays with {lvms}
```

Comment:
- For modules, the anchor ID must contain the {context} variable. This is the more important part of my comment and must be fixed before merge. This is because the `{context}` variable is inherited from the assembly, and modules can appear in different assemblies. [source](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)
- This heading/module title is different from the file name and ID; all three should match as closely as possible. I do understand that the `lvms-` prefix is standing in for an `-lvms` suffix in the file name and ID (we do a similar thing with our `virt-` prefix at times to avoid excessive repetition). However, the module title is now `= Integrating software RAID arrays with {lvms}`, which that diverges enough from the other two that I think it's worth trying to make them all match.

For example:

File name: `modules/lvms-using-raid-arrays.adoc`
ID: `[id="lvms-using-raid-arrays_{context}"]`
Title: `= Using RAID arrays with {lvms}`

or 

File name: `modules/lvms-integrating-software-raid-arrays.adoc`
ID: `[id="lvms-integrating-software-raid-arrays_{context}"]`
Title: `= Integrating software RAID arrays with {lvms}`

Reply:
Hi @dfitzmau 
We had a discussion on the usage of {context} variable in the IDs. 
Could you PTAL at this suggestion?
Thanks.

Reply:
Oh, sorry for the confusion, @sr1kar99 . I meant:

File name: modules/lvms-integrating-software-raid-arrays.adoc
ID: [id="lvms-integrating-software-raid-arrays_{context}"]
Title: = Integrating software RAID arrays with LVMS

Potentially swap `{lvms}` to  `LVM Storage`.

SOmetimes the customer portal can be funny with attributes in headings (=,==,===)

Was this clearer? Pan is correct about the context variable and its importance for document structuring.

Reply:
@dfitzmau 
I am sorry for the confusion. I understood it wrong. 
Added {context} variable in the ID, and updated {lvms} to LVM Storage in the heading.
@ousleyp, PTAL!
Thanks.